test/state-indicators.service.spec.js

```diff
@@ -116,6 +125,26 @@ describe('state indicators service', function () {
         }
       }
     })
+    spyOn(productListService, 'relevant').and.callFake(function () {
```

Comment:
Can you try mocking this at a module level like it's done in here https://github.com/fielded/angular-nav-state-indicators/blob/master/test/state-indicators.service.spec.js#L73 for the other two `angular-nav-data` services? I think that will solve your problem with the tests